Assets/Mirage/Runtime/ServerObjectManager.cs

```diff
@@ -59,8 +59,8 @@ public void Start()
 
                 if (NetworkSceneManager != null)
                 {
-                    NetworkSceneManager.ServerChangeScene.AddListener(OnServerChangeScene);
-                    NetworkSceneManager.ServerSceneChanged.AddListener(OnServerSceneChanged);
+                    NetworkSceneManager.ServerStartedSceneChange.AddListener(OnServerChangeScene);
+                    NetworkSceneManager.ServerFinishedSceneChange.AddListener(OnServerSceneChanged);
```

Comment:
please can you rename the methods for these too

Reply:
Rename them to what?